.github/workflows/delayed-pipelines.yml

```
@@ -0,0 +1,164 @@
+  build:
+    name: 🛠️ Build
+    runs-on: ubuntu-22.04
+    timeout-minutes: 15
+    strategy:
+      matrix:
+        configuration: [Debug, Release]
+        framework: [net9.0, net8.0, netstandard2.0]
+    needs: [prepare_linux]
+    outputs:
+      version: ${{ steps.minver-calculate.outputs.version }}
+      testProjects: ${{ steps.test-projects.outputs.result }}
```

actionlint

property "test-projects" is not defined in object type {dotnet-build: {conclusion: string; outcome: string; outputs: {string => string}}; minver-calculate: {conclusion: string; outcome: string; outputs: {string => string}}}

(expression)